public/js/monitor.js

```diff
@@ -410,6 +438,34 @@ function addBentoObserver(){
       });
     }
   }
+  const researchPromo = document.querySelector(".research-recruitment-wrapper");
+  const sendRecruitmentPing = (action, interactionStatus) => {
+    if (gaAvailable) {
+      ga("send", "event", "Research Recruitment Promo", action, "Recruitment Promo", {nonInteraction: interactionStatus});
+    }
+  };
+  const promoHasAlreadyBeenDismissed = (researchPromo && sessionStorage.getItem("fxMonitorResearchPromo") === "dismissed");
+  const sessionStorageDoesNotExist = (researchPromo && !sessionStorage);
+  const acceptedLanguages = navigator.languages;
+  const acceptedLocalesIncludesEnglish =
+    (acceptedLanguages.includes("en-US") || acceptedLanguages.includes("en"));
+
+  if (promoHasAlreadyBeenDismissed || sessionStorageDoesNotExist || !acceptedLocalesIncludesEnglish) {
+    researchPromo.style.display = "none";
```

Comment:
Could we save a lot of `researchPromoIsVisible` logic if we `return;` here?

Reply:
We could! Thank you!